<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="channel0_out_0">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="channel0_out_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="channel0_out_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="channel0_out_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="channel0_out_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="channel0_out_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="channel0_out_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="channel0_out_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 7:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>68</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Start:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>68</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Current:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X Scan Vector:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Scan Vector:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Pixels:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Settle time per pixel:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="start_x">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="start_y">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="start_z">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="start_aux">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="current_aux">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>60</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="current_y">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="current_z">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="current_x">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>68</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>x-axis</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>68</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>y-axis</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>68</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>z-axis</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>68</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>auxiliary</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="scanx_aux">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="scanx_y">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="scanx_z">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="scanx_x">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="scany_aux">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>180</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="scany_y">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="scany_z">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>180</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="scany_x">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="pixels">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="tsettle">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="setButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scanButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="abortButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Abort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>setButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scanButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>scanButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>abortButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>abortButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>782</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>779</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setButtonClicked()</slot>
  <slot>scanButtonClicked()</slot>
  <slot>abortButtonClicked()</slot>
  <slot>resetButtonClicked()</slot>
 </slots>
</ui>
